### ISSUE TITLE
Некорректный результат поиска по несуществующим словам на сервисе Здоровье.Мэйл.ру

### ISSUE THREAD

**Issue by Yamortis** [Yamortis/Test_health.mail.ru#7]
**Шаги:**
1. В браузере открыть страницу сервиса https://health.mail.ru/
2. В строке поиска ввести несуществующее слово (например: "инстомация")
3. Нажать кнопку поиска (лупу).

**Ожидаемый результат:**
ничего не найдено.

**Фактический результат:**
поиск выдает список учреждений https://yadi.sk/i/CIRta1tjTwksCQ

**Comment by Svetlana678** [comment]
Да, ошибка. По шагам - всё воспроизводится. Можно добавить рекомендацию по улучшению или указать в ожидаемом результате: например, появление фразы "Совпадений не найдено" или "Ничего не найдено".

**Comment by OlgaKondratieva** [comment]
Это не баг, а особенности настроек поиска. настроен так чтобы искать не все слово целиком а сочетание букв.
Обратите внимание на надпись после поиска: "Нашлось 2 материала. Возможно, вы имели в виду «интонация»?"
То есть он как бы предугадывает, а вдруг вы ошиблись набирая текст и предлагает наиболее подходящий вариант.
